servers/Azure.Mcp.Server/src/Azure.Mcp.Server.csproj

```diff
@@ -12,7 +12,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <AssemblyName>$(CliName)</AssemblyName>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
This seems like a substantial change, any reason behind it?

Reply:
Will turn back on before merge